controllers/comments.controllers.js

```diff
@@ -19,9 +19,19 @@ exports.postCommentsByArticleId = (req, res, next) => {
     const newComments = req.body
     addComments(article_id, newComments)
         .then((comments) => {
-        console.log(comments)
         res.status(201).send({comments})
     })
     .catch(next)
 }
 
+exports.deleteCommentByCommentId = (req, res, next) => {
+    const { comment_id } = req.params
+    const commentPromises = [deleteComments(comment_id)]
+    if(comment_id){commentPromises.push(checkCommentIdExists(comment_id))}
```

Comment:
this if statement isn't really needed.

Reply:
I seem to still get a 204 response with comment_id =99 without the before adding commentid check